Spigot-Server-Patches/0723-Fix-plugin-loggers-on-server-shutdown.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jos=C3=A9=20Miguel=20Moreno?= <josemmo@pm.me>
```

Comment:
The hell is this author-name

Reply:
https://en.wikipedia.org/wiki/MIME#Encoded-Word